### DISCUSSION TITLE
Organize replays into folders

### DISCUSSION THREAD

**Discussion by ceceliacreates** [RecordReplay/devtools#4947] [Ideas]
User request:

"If I had one request around collaboration it would be to be able to organize replays into folders. My second request would be the ability to share those folders to a specified subset of team members."

**Comment by jonbell-lot23** [comment]
March 15, 2022:

I think folders are definitely a part of our future. As we scale up, we'll need to add all sorts of things to the library like pagination and folders to make sure it's easy to move around even with lots of replays.
